xml/depl_nodes.xml

```diff
@@ -6479,7 +6482,197 @@ Apply the &barcl; to a Control Node.
    credentials for the &ostack; &dash; are user name <literal>admin</literal>
    and password <literal>crowbar</literal>.
   </para>
- </sect1>
+  </sect1>
+
+  <sect1 xml:id="crow-ses-integration">
+   <title>&ses; integration</title>
+   <para>
+    &productname; supports integration with &ses; (SES), enabling &ceph; block
+    storage as well as image storage services in &cloud;.
+   </para>
+   <bridgehead renderas="sect2">Enabling SES Integration
+   </bridgehead>
+   <para>
+    Start enabling SES integration by uploading a SES configuration file to
+    &crow;. This file is included in the <literal>crowbar-core</literal>
```

Comment:
It's not the "file" which is included in crowbar-core but the upload functionality. I think we don't really need the first party of the sentence here. It's pretty much an implementation detail.